engines/standard_tasks/app/models/standard_tasks/paper_reviewer_task.rb

```diff
@@ -6,6 +6,18 @@ def self.permitted_attributes
 
     register_task default_title: "Assign Reviewers", default_role: "editor"
 
+    include Invitable
+
+    def invitation_invited(invitation)
+      PaperReviewerMailer.delay.notify_invited({
+        invitation_id: invitation.id
+      })
+    end
+
+    def invitation_accepted(invitation)
+      TaskRoleUpdater.new(self, invitation.invitee_id, PaperRole::REVIEWER).update
+    end
+
     def array_attributes
       super + [:reviewer_ids]
```

Comment:
Yeah, the `Invitable` thing is new. Not necessarily the way I would have liked to do it, but the generically customizable tasks sorta tie our hands. Requires **so much** indirection.

I'm imagining paper role being more of a first class citizen, having a defined API boundary, as opposed to shimming everything into task. For the `reviewer_ids=` side-effect specifically, I think that's just a generic thing we should always try to avoid. I know it was there before you started working on it, so it may be out of scope this time.

Perhaps a a chore card for now now? See https://github.com/Tahi-project/tahi/pull/865/files for what I did.